cli/tools/init/mod.rs

```diff
@@ -53,10 +53,13 @@ Deno.test(function addTest() {
       "deno.json",
       &json!({
         "name": project_name,
-        "version": "1.0.0",
+        "version": "0.1.0",
```

Comment:
This is a better starting version IMO because you rarely release a `1.0.0` version the first time.